### PR TITLE
Add missing "static" modifier for constants

### PR DESCRIPTION
**Pull request by jukzi**
To reduce memory per instance

### CODE REVIEW

**Review by mpalat — APPROVED** [review]

**Comment by mpalat** [comment]
@jukzi :  +1 approving the code change after review. Please recheck the version bump with the latest before you merge.

**Comment by jukzi** [comment]
meanwhile the version bump was already submitted with another PR

**Pull request merged**
